functions/Export-DbaLogin.ps1

```diff
@@ -313,6 +313,29 @@ function Export-DbaLogin {
 					$credentialName = $credential.Name
 					$outsql += "PRINT '$userName is associated with the $credentialName credential'"
 				}
+
+				# Endpoint permissions
+				foreach ($endpoint in $server.endpoints) {
+                    $perms = $endpoint.EnumObjectPermissions($userName)
+
+                    foreach ($perm in $perms) {
+                        $permState = $perm.permissionstate
+                        $permType = $perm.PermissionType
+                        $grantor = $perm.grantor
+
+                        if ($permState -eq "GrantWithGrant") {
+                            $grantWithGrant = "WITH GRANT OPTION"
+                            $permState = "GRANT"
+                        }
+                        else {
+                            $grantWithGrant = $null
+                        }
+
+                        $outsql += "$permState $permType ON ENDPOINT::[$($endpoint.Name)] TO [$userName] $grantWithGrant AS [$grantor]"
```

Comment:
We should go ahead and add a statement terminator on the end `;` and then do we need to add a `GO` separator like we do in the other export functions @ConstantineK @ClaudioESSilva ?

Reply:
The `GO `is already there @ https://github.com/maddhatter/dbatools/blob/355ce02db5e718a41a8d1ec25006242c0fd08c23/functions/Export-DbaLogin.ps1#L396

The `;` I think is not being used in all places of the functions. But I agree we should add it.